docs/governance/overview.md

```diff
@@ -19,54 +19,94 @@ This page provides an overview of the On-chain Governance module that will be av
+- **Anyone** can create a proposal by paying the proposal fee and specifying:
+  - a **commit hash** (unique identifier, usually a GitHub commit or spec reference)
+  - a **start epoch** and an **end epoch** (the voting window).
+- **Any staked user** (direct or delegated) can vote during the active period.
+- Voting power is proportional to stake:  
+  `voting_power = staked_value` (linear voting).
 
 [comment]: # (mx-context-auto)
 
 ## Implementation details
 
-Each proposal costs 1000 EGLD that will be locked during the voting of the proposal. If the proposal is either rejected or accepted, the entire locked sum can be unlocked & withdrawn. If the proposal do not pass, 10 EGLD will remain locked in the governance smart contract as a penalty fee. The proposal costs around 51 million of gas units.
+### Proposals
 
-There are 4 types of votes: **Yes**, **No**, **Abstain** and **Veto**. Each of the vote costs around 6 million of gas units.
+In order for a proposal to be submitted, the following requirements need to be met:
+- For a period of at least 15 days, the proposal needs to be published on Agora for public debate and analysis;
+- Each proposal requires paying a `ProposalFee` (currently **500 EGLD**);
+- Each proposal costs around 51 million of gas units to be submitted.
+- The starting epoch of the proposal's voting period needs to be set inside an interval of 30 epochs from the epoch in which the proposal was submitted;
 
-A user can create any number of proposals as long as it pays the locking fee & the gas used for the transaction. The proposal can be closed in any epoch following the provided end epoch.
+After a proposal is created, the following rules will apply: 
+- If the proposal passes or fails normally, the fee is refunded to the issuer.  
+- If the proposal is vetoed, the fee is slashed (transferred to the **Community Governance Pool**) or reduced by the configured `LostProposalFee`.  
+- Proposals cannot be duplicated: the same commit hash cannot be submitted twice.
+- Proposals can be **cancelled** by the issuer **before the voting period starts**. (introduced in v1.10.0 Barnard release)
+- Lost proposal fees may either be accumulated in the governance contract (retrievable by the contract owner) or redirected to a **Community Governance Pool**, depending on configuration.  
 
-The quorum is computed as the sum of the staked EGLD for all addresses that cast votes.
 
-The votes will be added for each category (**Yes**, **No**, **Abstain** and **Veto**). The vote is computed as `vote = total_staked_value` for each address that cast a vote.
+### Voting
+- The voting starts from the provided starting epoch and lasts exactly *10 days*.
+- There are four vote types: **Yes**, **No**, **Abstain**, and **Veto**.  
+- Voting consumes gas (approx. 6 million units).  
+- Voting power is derived from staked and delegated EGLD.  
+- Delegation and liquid staking () contracts can cast votes on behalf of users.
```

Comment:
do we need the `()` ?
can cast votes on behalf of users -> can forward user votes. 

can cast votes on behalf sounds like the users can't/don't have to to anything.